### PR TITLE
added priority subcommand

### PR DESCRIPTION
**Pull request by Dotnester**

### CODE REVIEW

**Comment by Kobzol** [comment]
Looks fine :) Please rebase over `main` to resolve conflicts and add documentation to `docs/jobs.md`.

You can build the documentation locally using `mkdocs serve`. You can install mkdocs in Python:
```
$ python3 -m pip install mkdocs
$ python3 -m mkdoc serve
```

**Comment by spirali** [comment]
Code looks good, please add tests as we have discussed. If it is too complicated I can resolve conflicts myself

File: tests/test_job.py [hunk 1]
```diff
@@ -499,4 +496,67 @@ def test_job_resubmit_all(hq_env: HqEnv):
...
+    hq_env.start_server()
+    hq_env.command(
+        [
+            "submit",
+            "--priority",
+            "1",
+            "--",
+            "bash",
+            "-c",
+            "echo `date` && sleep 1",
```
Comment (Kobzol) [inline tests/test_job.py:512]:
```suggestion
            "date --iso-8601=seconds && sleep 1",
```

File: tests/test_job.py [hunk 2]
```diff
@@ -499,4 +496,67 @@ def test_job_resubmit_all(hq_env: HqEnv):
...
+    wait_for_job_state(hq_env, 2, "FINISHED")
+    wait_for_job_state(hq_env, 3, "FINISHED")
+    wait_for_job_state(hq_env, 4, "FINISHED")
+
+    dates = []
+    for file in ["stdout.1.0", "stdout.2.0", "stdout.3.0", "stdout.4.0"]:
+        with open(os.path.join(tmp_path, file)) as f:
+            parts = f.read().split(":")
+            assert len(parts) == 3
+            dates.append(int(parts[2][0])*10+int(parts[2][1]))
```
Comment (Kobzol) [inline tests/test_job.py:558]:
```suggestion
            dates.append(datetime.fromisoformat(f.read().strip()))
```

**Review by Kobzol — CHANGES_REQUESTED** [review]

**Comment by Kobzol** [comment]
@Dotnester thanks!

**Pull request merged**